transcendence-app/src/chat/chat.gateway.ts

```diff
@@ -132,6 +132,9 @@ export class ChatGateway {
       client.request.user.id,
       chatroomId,
     );
+    if (!crm) {
```

Comment:
before, there was some code path where we could be throwing an http exception here. Now it should happen, even conceptually, anymore. I'm not sure what would happen if we tried http exceptions in ws tbh.